core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3980,6 +3980,27 @@ paths:
                 $ref: '#/components/schemas/ErrorReport'
         500:
           $ref: '#/components/responses/RawlsInternalError'
+  /api/workspaces/v2/bucketMigration:
+    get:
+      tags:
+        - workspaces_v2
+      summary: get workspaces eligible for bucket migration and their migration progress if it exists
+      description: get workspaces eligible for bucket migration and their migration progress if it exists
+      operationId: getEligibleWorkspacesForBucketMigrationWithProgress
+      responses:
+        200:
+          description: Successful Request
+          content:
+            'application/json':
+              schema:
+                type: array
+                items:
+                  type: object
+                  description: workspaceNamespace/workspaceName -> bucket migration progress
+                  additionalProperties:
+                    $ref: '#/components/schemas/MultiregionalBucketMigrationProgress'
```

Comment:
So this progress reflects only the last migration attempt? (I'm reviewing the other PR and realize that one get can get an array of `WorkspaceMigrationMetadata` per workspace)

Reply:
That's correct, it's just the most recent attempt. The array of `WorkspaceMigrationMetadata` is a bit of carryover from PPW. The way we're doing restarts for this migration makes it far less likely for there to actually be multiple distinct migration attempts. In PPW, calling the `startMigration` API a second time would trigger a new distinct migration attempt. For this migration, calling the `startMigration` API a second time will attempt to restart the previous migration attempt